vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt

```diff
@@ -1,14 +1,29 @@
 package at.asitplus.wallet.lib.data
 
+import at.asitplus.wallet.lib.JsonValueEncoder
+import at.asitplus.wallet.lib.data.Json.serializersModules
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
 import kotlinx.serialization.modules.SerializersModule
 import kotlinx.serialization.modules.polymorphic
 import kotlinx.serialization.modules.subclass
 
-private val serializersModules = mutableMapOf<ConstantIndex.CredentialScheme, SerializersModule>()
+object Json {
```

Comment:
I know we have this in all our libs, but maybe we want to use a more expressive name? Not sure…